### PR TITLE
Englishization: editor

### PR DESCRIPTION
**Pull request by singlecoder**
- [x] Mars
- [x] SpriteRenderer
- [x] SpriteMask
- [x] Sprite

### CODE REVIEW

File: docs/editor-mars.en-US.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Usage
+
+Before using our engine to play Mars special effects, designers need to use Mars to make special effects and export the `.vfx` file. If you don’t know Mars, please take a look at [here](https://render.alipay.com/p/s/mars-editor/#).
+
+1. After the developer gets the `.vfx`, first upload the `.vfx` file to our Oasis editor, as follows:
+
+![image.png](https://gw.alipayobjects.com/mdn/rms_d27172/afts/img/A*lhieRKWLn64AAAAAAAAAAAAAARQnAQ)
+
+2. Select a entity and add the Mars component as follows:
```
Comment (yangfengzzz) [inline docs/editor-mars.en-US.md:17]: Select an entity

File: docs/editor-sprite-renderer.en-US.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+order: 8
+title: SpriteRenderer Component
+type: Editor
+---
+
+The sprite renderer component is used to display images in a 3D/2D scene, see [Sprite component](${docs}sprite-renderer-cn) for details.
+
+## Add SpriteRenderer component
+
+When you need to display a image, you need to add a sprite renderer component to an entity first, as follows:
```
Comment (yangfengzzz) [inline docs/editor-sprite-renderer.en-US.md:11]: display an image

File: docs/editor-sprite.en-US.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+---
+order: 8
+title: Sprite Resource
+type: Editor
+---
+
+[Sprite](${api}core/Sprite) is a 2D graphic object used for characters, props, bullets and some other 2D game elements. These graphics are obtained from [Texture2D](${docs}texture-cn). The [Sprite](${api}core/Sprite) class mainly identifies the part of the image applied to a specific `Sprite`. Then the [SpriteRenderer](${docs}sprite-renderer-cn) component on [Entity](${docs}entity-cn) can use this information to actually display graphics, [Entity](${docs}entity-cn ) On the [SpriteMask](${docs}sprite-mask-cn) component can use this information to represent the mask area. See [Spirite Resources](${docs}sprite-cn) for details.
```
Comment (yangfengzzz) [inline docs/editor-sprite.en-US.md:7]: [Sprite Resources]

File: docs/editor-sprite-renderer.en-US.md (hunk 2 above)
Comment (yangfengzzz) [reply docs/editor-sprite-renderer.en-US.md:11]: It's better: When you need to display a image, a sprite renderer component should be added to an entity first.

use more passive words.

File: docs/editor-sprite-mask.en-US.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+order: 8
+title: SpriteMask Component
+type: Editor
+---
+
+The sprite mask component is used to mask the [sprite](${docs}editor-sprite-renderer-cn) in the 3D/2D scene. For details, please refer to the [sprite mask component](${docs}sprite-mask- cn).
+
+## Add sprite mask component
+
+When we need to mask a sprite, we first need to create an entity and add the sprite mask component, as follows:
```
Comment (yangfengzzz) [inline docs/editor-sprite-mask.en-US.md:11]: When we need to mask a sprite, an new entity with the sprite mask component can be created as follows:

more passive words.

File: docs/editor-mars.en-US.md (hunk 1 above)
Comment (singlecoder) [reply docs/editor-mars.en-US.md:17]: > Select an entity

+1

File: docs/editor-sprite-renderer.en-US.md (hunk 2 above)
Comment (singlecoder) [reply docs/editor-sprite-renderer.en-US.md:11]: done

File: docs/editor-sprite.en-US.md (hunk 3 above)
Comment (singlecoder) [reply docs/editor-sprite.en-US.md:7]: done

File: docs/editor-sprite-mask.en-US.md (hunk 4 above)
Comment (singlecoder) [reply docs/editor-sprite-mask.en-US.md:11]: done

**Pull request closed without merging**